src/components/NavBar/ChainSelector.tsx

```diff
@@ -72,6 +70,12 @@ export const ChainSelector = ({ leftAlign }: ChainSelectorProps) => {
 
   const theme = useTheme()
 
+  const [chains, setChains] = useState(NETWORK_SELECTOR_CHAINS)
```

Comment:
this could be a `useMemo` instead although i'm not sure it brings any benefits. `useMemo` may be simpler to parse at first glance since we don't have to go find all the `setChains` callsites (in this case only 1 but reader doesn't know that)